utils.js

```diff
@@ -73,7 +73,13 @@ module.exports = {
         var kvArray = str.split('&');
         for(var j = 0; j < kvArray.length; j++){
           var kvString = kvArray[j].split('=');
-          obj[kvString[0]] = kvString[1];
+          if (Array.isArray(obj[kvString[0]])) {
+            obj[kvString[0]].push(kvString[1])
+          } else if (obj[kvString[0]]) {
+            obj[kvString[0]] = [obj[kvString[0]], kvString[1]];
+          } else {
+            obj[kvString[0]] = kvString[1];
+          }
```

Comment:
Thank you, this addition looks great! Seeing as the code added to both `arrayToObj` and `stringToObj` is identical, would you mind moving them into a separate method? Once that's in place, I'll be happy to merge.

Reply:
Here it is. I prefer the functional programming paradigm, so I'm a bit leery of extracting into a function that performs mutation by reference. But `Object.assign` wasn't implemented in ES5 and I doubt you'd appreciate me bloating your library w/ a polyfill or lodash.

I did write the function so it has a return value and added a warning in the declaration comments to mitigate--will that work.